spec/helpers/comments_helper_spec.rb

```diff
@@ -0,0 +1,194 @@
+# frozen_string_literal: true
+
+describe CommentsHelper, type: :controller do
+  let(:user1) { create(:user1) }
+  let(:user2) { create(:user2) }
+
+  controller(ApplicationController) do
```

Comment:
Even though we don't run Rubocop on test files, I think you can do `controller(ApplicationController) {}` if you want to keep it as a one-liner

Reply:
Gotcha! Not that I think we will put anything in that block, but I'm going to leave it two lines to keep consistent with how RSpec documents it.